lib/auth/iam.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2013 Google Inc. All Rights Reserved.
```

Comment:
Is there a policy for the year on the file?  I added a lint rule to verify that we have the header, and it includes the year.  I was hoping to make them consistent.